index.html

```diff
@@ -51,7 +51,70 @@ <h2 class="describe">I’am a Software Developer</h2>
         </ul>
```

Comment:
- [ ] Kindly make sure your nav bar elements are well centered as the design in Figma.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198566366-b303b977-ba1d-4013-a06f-ae691b33bf5f.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198566517-3bb80342-67bf-4684-999d-c93aff4febab.png)